### PR TITLE
Supporting env var assignments on Windows

### LINKED ISSUE #313: Support Windows env var assignments
Per #304, we do not support Windows env var assignments.  We should probably use [cross-env](https://www.npmjs.com/package/cross-env) for this.

``` sh
npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Joshua\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v4.4.3
npm ERR! npm  v3.10.2
npm ERR! code ELIFECYCLE
npm ERR! stardust@0.16.4 prepublish: `NODE_ENV=production npm run build:dist`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the stardust@0.16.4 prepublish script 'NODE_ENV=production npm run build:dist'.
```

### PR DESCRIPTION
**Pull request by joshuazmiller**
Trying to resolve #313

### CODE REVIEW

**Comment by joshuazmiller** [comment]
Can we ignore the failed coverage test? You can't write coverage to the best of my knowledge.

File: package.json [hunk 1]
```diff
@@ -13,19 +13,19 @@
...
-    "prepublish": "NODE_ENV=production npm run build:dist",
+    "prepublish": "cross-env NODE_ENV=production npm run build:dist",
     "release:major": "ta-script npm/release.sh major",
     "release:minor": "ta-script npm/release.sh minor",
     "release:patch": "ta-script npm/release.sh patch",
     "start": "npm run docs",
     "start:local-modules": "npm run docs -- --local-modules",
     "pretest": "gulp dll",
-    "test": "NODE_ENV=test babel-node $(npm bin)/karma start build/karma.conf.babel.js",
+    "test": "cross-env NODE_ENV=test babel-node node_modules/karma start build/karma.conf.babel.js",
```
Comment (levithomason) [inline package.json]: This should be `node_modules/.bin/karma`.

Details:
When you `npm install` packages that come with executables, they are placed in `node_modules/.bin`.  You can get the absolute path to the npm bin with the command `npm bin`.  NPM also adds this directory to your `$PATH` when you `npm run ...` scripts.  This way you can execute scripts that you have installed.  The `$(npm bin)` syntax is command substitution.  It places the output of that command inline in another command.  So, we were placing the absolute path to the npm bin directory just before `/karma`.

**Comment by levithomason** [comment]
Thanks for the PR.  Please reference the related issue in the description in order to [close the issue when the PR is merged](https://github.com/blog/1506-closing-issues-via-pull-requests).

Coverage will not fail for this PR.  Coverage is calculated by lines of code covered by tests.  Since this PR touches no code, the coverage will not be affected.  CircleCI seems to have blown up for some random reason.  It should run on the next commit.

**Comment by levithomason** [comment]
@joshuazmiller it looks like tests are failing because they are never running in the first place. Pretty sure this is due to the karma path issue I noted here https://github.com/TechnologyAdvice/stardust/pull/316#discussion_r68971089

File: package.json (hunk 1 above)
Comment (joshuazmiller) [inline package.json]: Thank you for explaining that to me. I learned something new!

**Comment by joshuazmiller** [comment]
It doesn't seem I have permission to manually close #313 , however I added it to my description and perhaps it will work now. I'll keep in mind to mark those in for future pull-requests.

When I run test with the .bin I get the following error:

> C:\Users\Joshua\Dropbox\GitHub\stardust\node_modules.bin\karma:2
> basedir=$(dirname "$(echo "$0" | sed -e 's,\,/,g')")
>           ^^^^^^^
> 
> SyntaxError: missing ) after argument list
>     at exports.runInThisContext (vm.js:53:16)
>     at Module._compile (module.js:373:25)
>     at Module._extensions..js (module.js:416:10)
>     at Object.require.extensions.(anonymous function) [as .js](C:UsersJoshuaDropboxGitHubstardustnode_modulesbabel-registerlibnode.js:166:7)
>     at Module.load (module.js:343:32)
>     at Function.Module._load (module.js:300:12)
>     at Function.Module.runMain (module.js:441:10)
>     at C:\Users\Joshua\Dropbox\GitHub\stardust\node_modules\babel-cli\lib_babel-node.js:160:24
>     at Object.<anonymous> (C:\Users\Joshua\Dropbox\GitHub\stardust\node_modules\babel-cli\lib_babel-node.js:161:7)
>     at Module._compile (module.js:409:26)

I haven't written bash scripts in years, and I pretty much can't make sense of the file to understand what's wrong.

**Comment by levithomason** [comment]
Ah, yes. The karma script that is running is not compatible with Windows. You would likely need to run the "karma.cmd" version instead.

We can't hard code the test script to that or it won't work on Unix machines. We'll need to figure out how to support Windows here. 

Any help appreciated as I so not have a Windows machine available. I will do some digging on this though.

**Comment by levithomason** [comment]
> It doesn't seem I have permission to manually close #313 

Indeed, even if you did have permissions the best practice is to leave the issue open until the fix is merged.  The description you have in this PR should take care of it!

**Comment by levithomason** [comment]
@joshuazmiller I just merged #320 which removes `babel-node` entirely.  This means we no longer need any special paths to `karma` on the test script.  I am hoping by running karma directly like this that `npm` runs the correct version for Windows and also for Unix.

Please update this PR with the latest master and set the `test` script to:

```
"test": "cross-env NODE_ENV=test karma start build/karma.conf.babel.js",
```

Let's see if tests then pass, and also if it runs locally for you.

**Comment by joshuazmiller** [comment]
Unfortunately, I still get errors after the merge and adding that line:

> cross-env NODE_ENV=test karma start build/karma.conf.babel.js
> 
> 30 06 2016 21:55:48.509:ERROR [config]: Invalid config file!
>   SyntaxError: Unexpected token {
>     at exports.runInThisContext (vm.js:53:16)
>     at Module._compile (module.js:373:25)
>     at Object.Module._extensions..js (module.js:416:10)
>     at Module.load (module.js:343:32)
>     at Function.Module._load (module.js:300:12)
>     at Module.require (module.js:353:17)
>     at require (internal/module.js:12:17)
>     at Object.parseConfig (C:\Users\Joshua\Dropbox\GitHub\stardust\node_modules\karma\lib\config.js:284:22)
>     at new Server (C:\Users\Joshua\Dropbox\GitHub\stardust\node_modules\karma\lib\server.js:57:20)
>     at Object.exports.run (C:\Users\Joshua\Dropbox\GitHub\stardust\node_modules\karma\lib\cli.js:243:7)
>     at requireCliAndRun (C:\Users\Joshua\Dropbox\GitHub\stardust\node_modules\karma-cli\bin\karma:44:16)
>     at C:\Users\Joshua\Dropbox\GitHub\stardust\node_modules\karma-cli\bin\karma:54:12
>     at C:\Users\Joshua\Dropbox\GitHub\stardust\node_modules\resolve\lib\async.js:44:21
>     at ondir (C:\Users\Joshua\Dropbox\GitHub\stardust\node_modules\resolve\lib\async.js:187:31)
>     at C:\Users\Joshua\Dropbox\GitHub\stardust\node_modules\resolve\lib\async.js:153:39
>     at onex (C:\Users\Joshua\Dropbox\GitHub\stardust\node_modules\resolve\lib\async.js:93:22)
> npm ERR! Windows_NT 10.0.10586
> npm ERR! argv "C:\Program Files\nodejs\node.exe" "C:\Users\Joshua\AppData\Roaming\npm\node_modules\npm\bin\npm-cli.js" "run" "test"
> npm ERR! node v4.4.7
> npm ERR! npm  v3.10.2
> npm ERR! code ELIFECYCLE
> npm ERR! stardust@0.16.4 test: `cross-env NODE_ENV=test karma start build/karma.conf.babel.js`
> npm ERR! Exit status 1

**Comment by levithomason** [comment]
I see in your error output you are running:

``` sh
npm ERR! node v4.4.7
npm ERR! npm v3.10.2
```

This project requires Node 6.  I just added [a note](https://github.com/TechnologyAdvice/stardust/blob/master/CONTRIBUTING.md#getting-started) about this in the contributing guide last night. 

Please install Node 6 and give it another go. Thanks for hanging in there through all this! I think we're close to a solution.

**Comment by levithomason** [comment]
@joshuazmiller did updating Node help at all?

**Comment by joshuazmiller** [comment]
It works now! Thanks!

**Comment by levithomason** [comment]
Awesome! 😸 Very glad we finally got it.

Just one last tweak and we can merge this PR.  It looks like the package description got set wrong (probably after a conflict fix).  It should be:

``` json
"description": "The official Semantic-UI-React integration."
```

Once that is updated, we can get this PR in.  Thank you very much for hanging in there.

**Comment by joshuazmiller** [comment]
This should do it. Thank you. I am going to get back to the List Item and Item API as soon as I can

**Pull request merged**